docs/userguide/networking/get-started-overlay.md

```diff
@@ -16,7 +16,6 @@ network. Docker Engine supports multi-host networking out-of-the-box through the
 `overlay` network driver.  Unlike `bridge` networks, overlay networks require
 some pre-existing conditions before you can create one. These conditions are:
 
-* A host with a 3.16 kernel version or higher.
```

Comment:
@wenchma that's because 3.10 is the minimum supported kernel version for docker engine. Pls refer to https://docs.docker.com/engine/installation/ubuntulinux/ and other distributions for the pre-req documentation. I guess this covers all the docker features including overlay networking.